README.md

```diff
@@ -143,7 +143,7 @@ this:
   CCP is reduced to 400px tall.
 * CSS styles you add to your site will NOT be applied to the CCP because it is
   rendered in an iframe.
-
+* If you are trying to use chat specific functionalities, please also include [ChatJs](https://github.com/amazon-connect/amazon-connect-chatjs) in your code. We keep Streams and ChatJs seperate because ChatJs is only needed when Streams is being used for agent chat.
```

Comment:
Overall this is good, but I would suggest changing the last sentence to: "We omit ChatJs from the Makefile so that streams can be used without ChatJS. Streams only needs ChatJS when it is being used for chat." Otherwise, someone could get confused and assume ChatJS is only ever needed to be used within streams. That is not the case, chatjs can be used for customer chats outside of streamsjs.

Reply:
I updated the text based on your comment.